src/riscv/etherlink/Makefile

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 TriliTech <contact@trili.tech>
+#
+# SPDX-License-Identifier: MIT
+
+.PHONY: all
+all: build test check
+
+.PHONY: build
+build:
+	@cargo build --release
+
+.PHONY: test
+test:
+	@# This executable compiles to RISC-V. It should be run in the RISC-V sandbox.
+
```

Comment:
Really no need to inherit this target if it does nothing.

Reply:
Removed now.